enterprise_access/apps/subsidy_request/tasks.py

```diff
@@ -27,3 +34,95 @@ def update_course_title_for_subsidy_request_task(subsidy_type, subsidy_request_u
 
     # Use bulk_update so we don't trigger save() again
     subsidy_model.bulk_update([subsidy_request], ['course_title'])
+
+
+def _get_manage_requests_url(subsidy_model, enterprise_slug):
+    """
+    Get a manage_requests url based on the type of subsidy.
+
+    Args:
+        subsidy_model (class):  class of the subsidy object
+        enterprise_slug (string): slug of the enterprise's name
+    Returns:
+        string: a url to the manage learners page.
+    """
+    if subsidy_model == apps.get_model('subsidy_request.LicenseRequest'):
+        subsidy_string = 'subscriptions'
+    else:
+        subsidy_string = 'coupons'
+
+    url = f'{settings.ENTERPRISE_ADMIN_PORTAL_URL}/{enterprise_slug}/admin/{subsidy_string}/manage-requests'
+    return url
+
+
+@shared_task(base=LoggedTaskWithRetry)
+def send_admins_email_with_new_requests_task(enterprise_customer_uuid):
+    """
+    Task to send new-request emails to admins.
+
+    Args:
+        enterprise_customer_uuid (str): enterprise customer uuid identifier
+    """
```

Comment:
You know it raises at least one exception. It should be part of the docs.

Reply:
This is a recurring theme where you ask me to put things in the docstrings. Do we have linting for that?

Reply:
I believe this is definitely a tech debt item, we do not have consistent documentation, and I think I actually logged it at one point.

Reply:
techdebt item added and put in a line for raises in latest commit